ui/components/fixture-capability-type-icon.vue

```diff
@@ -0,0 +1,189 @@
+      iconProps.colors = [cap.shutterEffect === `Closed` ? `#000000` : `#ffffff`];
+      delete iconProps.name;
+    }
+    else {
+      iconProps.name = `Strobe`;
+    }
+  },
+  ColorIntensity(cap, iconProps) {
+    iconProps.name = `Intensity`;
+  },
+  ColorPreset(cap, iconProps) {
+    iconProps.name = `Color`;
+  },
+  PanContinuous(cap, iconProps) {
+    if (cap.speed[0].number === 0 && cap.speed[1].number === 0) {
+      iconProps.name = `SpeedStop`;
+    }
+    else if (cap.speed[0].number > 0 || cap.speed[1].number > 0) {
+      iconProps.name = `pan-continuous-cw`;
+    }
+    else if (cap.speed[0].number < 0 || cap.speed[1].number < 0) {
+      iconProps.name = `pan-continuous-ccw`;
+    }
+  },
+  TiltContinuous(cap, iconProps) {
+    if (cap.speed[0].number === 0 && cap.speed[1].number === 0) {
+      iconProps.name = `SpeedStop`;
+    }
+    else if (cap.speed[0].number > 0 || cap.speed[1].number > 0) {
+      iconProps.name = `tilt-continuous-cw`;
+    }
+    else if (cap.speed[0].number < 0 || cap.speed[1].number < 0) {
+      iconProps.name = `tilt-continuous-ccw`;
+    }
+  },
+  WheelSlot(cap, iconProps) {
+    const isAnimationGobo = slot => slot.type.startsWith(`AnimationGobo`);
+    if (isAnimationGobo(cap.wheelSlot[0]) && isAnimationGobo(cap.wheelSlot[1])) {
+      iconProps.name = `Gobo`;
+    }
+    else if (cap.wheelSlot[0] === cap.wheelSlot[1] && cap.wheelSlot[0].type !== `Split`) {
+      iconProps.name = cap.wheelSlot[0].type;
+    }
+    else {
+      iconProps.name = ``;
+    }
+  },
+  WheelShake(cap, iconProps) {
+    iconProps.name = (cap.isShaking === `slot`)
+      ? `SlotShake`
+      : `WheelShake`;
+  },
+  IrisEffect(cap, iconProps) {
+    iconProps.name = `Iris`;
+  },
+  FrostEffect(cap, iconProps) {
+    iconProps.name = `Frost`;
+  },
+  Fog(cap, iconProps) {
+    iconProps.name = cap.fogType || `Fog`;
+  },
+  FogOutput(cap, iconProps) {
+    iconProps.name = `Fog`;
+  },
+  FogType(cap, iconProps) {
+    iconProps.name = cap.fogType;
+  },
+  Speed(cap, iconProps) {
+    if (!cap.speed) {
+      iconProps.name = `speed`;
+    }
+    else if (cap.speed[0].number === 0 && cap.speed[1].number === 0) {
+      iconProps.name = `SpeedStop`;
+    }
+    else if (cap.speed[0].number > 0 || cap.speed[1].number > 0) {
+      iconProps.name = `speed-forward`;
+    }
+    else {
+      iconProps.name = `speed-reverse`;
+    }
+  },
+  Rotation(cap, iconProps) {
+    if (cap.speed) {
+      specialIconFunctions.RotationSpeed(cap, iconProps);
+    }
+    else {
+      specialIconFunctions.Angle(cap, iconProps);
+    }
+  },
+  RotationSpeed(cap, iconProps) {
+    if (cap.speed[0].number === 0 && cap.speed[1].number === 0) {
+      iconProps.name = `SpeedStop`;
+    }
+    else if (cap.speed[0].number > 0 || cap.speed[1].number > 0) {
+      iconProps.name = `rotation-cw`;
+    }
+    else {
+      iconProps.name = `rotation-ccw`;
+    }
+  },
+  Angle(cap, iconProps) {
+    iconProps.name = `angle`;
```

Comment:
Is this really needed? Just replace the above method call with `` iconProps.name = `angle` ``. And maybe this icon should actually be called `rotation-angle` as the existing rotation angle icon is unused?

Reply:
You're right, this can be simplified. I think I had some more sophisticated logic for angle icons in mind. Which icon do you like better – `angle` or `rotation-angle`? In either case, we can delete the other one and rename the remaining one to `rotation-angle`.

Reply:
You're right, this can be simplified. I think I had some more sophisticated logic for angle icons in mind. Which icon do you like better – `angle` or `rotation-angle`? In either case, we can delete the other one and rename the remaining one to `rotation-angle`.

Reply:
I'd prefer the current `angle` icon, renamed to `rotation-angle`. It better indicates that it's not about an opening angle (i.e. beam angle).